CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to grammY Tools
```

Comment:
grammY Tools? Did we just rebrand?

Reply:
I thought we were working on grammY tools, not grammY Tools. \*confused\*

Reply:
are talking about the uppercase?

Reply:
Turns out to be yes.

Reply:
Convention. See Telegram private messages.

Reply:
So far, all repos use _grammY lowercase_ as names so grammY Tools would be the exception here.

Reply:
Jess, jou are right!